### ISSUE TITLE
Cannot load library /opt/SQLiteStudio/plugins/libDbSqliteCipher.so

### ISSUE THREAD

**Issue by dkoudstaal** [pawelsalawa/sqlitestudio#3856]
### Details
After installation in the /opt directory I get the following error:
[17:22:41] Cannot load plugin DbSqliteCipher. Error details: Cannot load library /opt/SQLiteStudio/plugins/libDbSqliteCipher.so: (libcrypto.so.1.0.0: cannot open shared object file: No such file or directory)

Same error message with the portable version

However the directory (install version) and the file is present in the /opt directory

### Steps to reproduce
Downloaded InstallSQLiteStudio-3.2.1 (installer) from https://github.com/pawelsalawa/sqlitestudio/releases home folder
Made the file executable
Accepted the default folder for installation.

Same issue with the portable version - directory and file exist in the directory in which the sqlitestudio-3.2.1.tar.xz (portable) archive was extracted

### Operating system
Using Ubuntu 20.04

### SQLiteStudio version
Version 3.2.1

**Comment by pawelsalawa** [comment]
Do `apt search libcrypto.so.1.0.0` - it will tell you what package has it. Then do `apt install that_package`

**Comment by dkoudstaal** [comment]
Thank you very much for your prompt reply. `apt search libcrypto.so.1.0.0` comes up as follows:
```
sudo apt search libcrypto.so.1.0.0
Sorting... Done
Full Text Search... Done
```
Using synaptic did not result in an option to install the library either.
After a bit of reading it seems the package is not available in Ubuntu 20.04
I found the following work around: In [Ask Ubuntu](https://askubuntu.com/questions/1261614/ubuntu-20-04-libssl-so-1-0-0-cannot-open-shared-object-file-no-such-file-or-d) Answer by flobede
insert the following line in `/etc/apt/sources.list` (at the end of the document is as good a place as any)
`deb http://security.ubuntu.com/ubuntu xenial-security main`
Save the file /etc/apt/sources.list 
Install after update `libssl1.0.0`: 
```
sudo apt update
sudo apt install libssl1.0.0
```
This solved the issue and there is no error any more!
